90.1 PRM heat pump data

AmmarDeh opened the pull request.

AmmarDeh committed:
Bump up maximum capacity of unitary air conditioners tables

Bump up maximum capacity of unitary air conditioners tables including 90.1, 90.1 PRM and IECC

AmmarDeh committed:
Expand 90.1 heat pump data

Updated 90.1 heat pump data including heating and cooling requirements.

AmmarDeh committed:
Add 90.1 PRM heat pump data

Added 90.1 heat pump efficiency requirements (for both heating and cooling)

lymereJ reviewed (changes requested):
@AmmarDeh - Overall it looks great. I think we do however need to add an `equipment_type` field, just like for the air-conditioner data. Otherwise, it might be hard to make the difference between some of the records other. Example of values: `Unitary and Applied Heat Pumps`, `Through the Wall`, etc.

AmmarDeh committed:
Modify heat pump prescriptive and PRM data

Added "equipment_type" to the database tables for both heating and cooling.
Added "rating_condition" as a string instead of numeric for the heat pump heating data.

AmmarDeh commented:
> @AmmarDeh - Overall it looks great. I think we do however need to add an `equipment_type` field, just like for the air-conditioner data. Otherwise, it might be hard to make the difference between some of the records other. Example of values: `Unitary and Applied Heat Pumps`, `Through the Wall`, etc.

Good catch! This is because the column `equipment_type` wasn't added to the HP database table (.py file). This should be fixed now!
